alembic/versions/1aeb080b0eda_galaxy_table.py

```diff
@@ -0,0 +1,62 @@
+"""galaxy table
+
+Revision ID: 1aeb080b0eda
+Revises: 80434b53562c
+Create Date: 2021-11-20 14:58:47.726104
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import healpix_alchemy
+
+# revision identifiers, used by Alembic.
+revision = '1aeb080b0eda'
+down_revision = '80434b53562c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'galaxys',
```

Comment:
This is an awkward plural. Not sure how we handle table naming in models/ but this table should be either `galaxy` or `galaxies`. Likewise the indices should use the proper plural.

Reply:
@acrellin @stefanv The "s" is adding automatically I guess. Any thoughts on this?

Reply:
It's a bit awkward either way -- we can force it to be `"galaxies"`, but then any associated join tables will have `"galaxie_id"` because of how the singular-plural-singular conversion works.  E.g. we forced the photometry table to be `"photometry"`, but now all join tables references that table use `"photometr_id"` which is pretty awkward.  I think `"galaxys"` is fine campared to `"galaxie_id"`. Thoughts?